src/js/lib/content/webpack.config.js

```diff
@@ -1,12 +1,20 @@
 const CopyWebpackPlugin = require('copy-webpack-plugin')
 const CompressionPlugin = require('compression-webpack-plugin')
 const path              = require('path')
+const webpack           = require('webpack')
 
 const thisPath = path.resolve(__dirname)
 const root     = path.resolve(thisPath,'..','..','..','..')
 const distPath = path.resolve(root,'dist')
 const wasmPath = path.resolve(distPath,'wasm')
 
+const child_process = require('child_process');
+function git(command) {
+    return child_process.execSync(`git ${command}`, { encoding: 'utf8' }).trim();
+}
+
+const BUILD_INFO = JSON.parse(require('fs').readFileSync('../../../../dist/build.json', 'utf8'));
```

Comment:
I don't like doing it this way because of few reasons:
1. This should be either in a separate section, or should have lovercase name, like other file path constants above.
2. Instead of using such string, it is always recommended to use `path.resolve` instead. 
3. Why do we use so many `../` in this string, if the `root` path is already resolved above?

Reply:
Simplified to make use of the `root`.